components/Feature/Services/index.js

```diff
@@ -40,6 +46,20 @@ const Services = ({
     }
   };
 
+  const sendFeedback = () => {
+    let enteredText = document.getElementById('search-feedback').value;
+    if (enteredText?.length > 0) {
+      sendDataToAnalytics({
+        action: getUserGroup(referrerData['user-groups']),
+        category: FEEDBACK_SEARCH,
+        label: document.getElementById('keyword-search').value,
```

Comment:
Hm same here maybe (?) have it as a hidden input or something (I feel uneasy about using getElementById where it's possible to avoid it cause it's DOM manipulation to an extent (?)). Happy to hear a different opinion

Reply:
Thanks - this highlighted an issue. I'd reintroduced showing the Search Feedback form when they narrow down results by category too. This meant it was showing the search input value when sending feedback in this scenario.

I've solved your comment by moving this to state, and set the same value when filtering by category.